lib/Service/DashboardService.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Talk\Service;
+
+use OCA\Talk\Exceptions\ParticipantNotFoundException;
+use OCA\Talk\Exceptions\RoomNotFoundException;
+use OCA\Talk\Manager;
+use OCA\Talk\Participant;
+use OCA\Talk\Room;
+use OCP\AppFramework\Utility\ITimeFactory;
+use OCP\Calendar\IManager;
+use Psr\Log\LoggerInterface;
+
+class DashboardService {
+	public function __construct(
+		private Manager $manager,
+		private IManager $calendarManager,
+		private ITimeFactory $timeFactory,
+		private LoggerInterface $logger,
+		private ParticipantService $participantService,
+	) {
+
+	}
+
+	/**
+	 * @param string $userId
+	 * @return Participant[]
```

Comment:
```suggestion
	 * @return list<Participant>
```

If psalm complains, we add a `array_values` on the return at the end.